### DISCUSSION TITLE
What's the Real Reason the CHIA-DEV God's Tell us NOT to Use HPOOL?

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#5837] [General]
There are lots of chia-explorers out there that integrate all the found new blocks on earth ever 5-15 minutes, they also report the number of plots submitted; So to answer your question the data comes from the chia-explorers, where they average all the plots submitted.

https://miningpoolstats.stream/chia

The majority of all plots are on HPOOL in China, +60% last time I looked then another 30% come from china as anonymous, but I think they're also hpool, then the remaining 10% come from the west.

Chia-NET (dev) doesn't have a clue how many plots are being' 'farmed' because NOBODY in china uses the Chia-SW for anything, HPOOL provides their own miner, and their own plotter, nobody gives a hell about the chia blockchain, probably one of the big reason it doesn't work is that the entire network is about 'mining' XCH, and not about transactions, or blockchain,

https://chiacalculator.com/

The entire idea of mining has been thrown out the window, its not about transactions, its all about the plot lottery.

All this BS about plots, farms, harvesters,  & farmers. Was a great 'woke' green narrative, the truth however is that its the same as it always is/was the rich are getting richer, and the poor are getting screwed.

Chia-Net can only estimate how many of its sucker (users) have submitted plots through its harvesting nodes and integrated value, given that is 10% of all plots submitted, that's why the real data is picked up from the explorers, there are also global pool websites that monitor all XCH found, and the submitted plots; all this data is integrated. IMHO CHIA-NET get their EB data from the chia-calc guy, who sells plots, this scam is one big happy family of scammers.

So why does Bram&Cohen tell its sheep, aka flock of cultists "Do use Hpool, you'll shoot your eye out", well its because they hate to see anybody but themselves making money.

Then there is the control thing, chia-net wants control, but that was lost 6+ months ago. Funny the God's complain about China owning the 'mining' (Farming) but they never complain about the IOU's where Chinese gamblers shot the CHIA XCH prices  from zero to $1600.

Cannot have it both ways "Gods of Chia".

As of this point chia is evolving into the software from hell;

Space is now slowing down from 8% a day to 3% a day, but the price is also dropping 3% a day, once the price of Chia nears zero the space will drop dramatically as when there is no longer profit in gambling chia with plots in china then 90% of the worlds plots are pulled out, but also 99% of chia's investment capital from China is also pulled out.

What is left is Bram&Cohen get their wish they get to control a dead product. Moving forward the Chinese will not come back, and the $$$ will not come from 'woke' Greeny's in the West.

Chia has still not 'purposed itself', so right now we have 8.8EB of wasted disk space filled up with random numbers for what? 51k new plots added per day at 108GB, for what? 

The guy in CHINA who ends up owning this thing by mining majority can deploy that EB anyway he wishes, he can rent the space out to anybody who wants to buy storage online just like Amazon and Google do, or file-chain.

Of course this has been Cohen's plan all along, remember he's the porn king of the earth 60% of the internet is BITTORRENT, and 90% of bittorrent is PORN. Don't think for a moment that Cohen losing control of the mining process doesn't destroy all his long term plans for renting out +10EB to the worlds CP content industry.

Moving forward who knows what's going to happen, but its not going to be chia-net, or anybody associated with these people running, and ruling CHIA Classic. It will certainly be China, an some billionaire will adopt it; The Chinese GOV actually likes file-coin, and I suspect they like Chia as well, this is why they will NOT let Cohen turn CHIA into a PORN server, as CCP hates porn, and prostituion, and CP, and child abuse, and denigration of women, this goes back to the founding of CCP, in the 1920's Shanghai was one big whore-house ran by the USA; They have never forgotten.

**Comment by chiabug** [comment]
You have a lot of points in your post.  Obviously you have spent some time thinking about Chia.  

You said "**its not about transactions, its all about the plot lottery**".  This is a point that I think is VERY important.  To make a truly sustainable "green" system the underlying "work" has to be something that has meaning and value -- such as the processing and storing of transactions.  A global immutable database has real long term value.  A system that is only processing transactions created by it's own self is just non-green busy work driven by speculation and/or greed.  To get Chia past this hurdle requires the software to be polished, reliable and useful in processing all types of transactions.

**Comment by mkdev-code** [comment]
its not farming just que with 200k(and some with fake plots) users in hpool seems
